webui/package.json

```diff
@@ -66,6 +67,7 @@
     "happy-dom": "^8.9.0",
     "react-test-renderer": "^18.2.0",
     "rollup-plugin-analyzer": "^4.0.0",
+    "sass": "^1.62.1",
```

Comment:
> This is another stylesheets package!?   

Another? Which other ones do we have? :) This is more of a build tool, to be specific.

It is not because of `usehooks-ts` and it's a `devDependancy`, so it doesn't affect the bundle size.